### PR TITLE
Modify climateiq_spatialize_chunk_predictions_cf based on live testing 

### PR DESCRIPTION
**Pull request by skeshive**

### CODE REVIEW

File: cloud_functions/climateiq_spatialize_chunk_predictions_cf/main.py [hunk 1]
```diff
@@ -200,12 +208,11 @@ def _chunk_metadata_fields_valid(chunk_metadata: dict) -> bool:
     )
 
 
-def _get_chunk_metadata(study_area_metadata: dict, chunk_id: str) -> dict:
+def _get_chunk_metadata(chunks_ref: Any, chunk_id: str) -> dict:
```
Comment (sylmak) [inline cloud_functions/climateiq_spatialize_chunk_predictions_cf/main.py:211]: Nit: Can this be typed more specifically? http://cloud/python/docs/reference/firestore/latest/google.cloud.firestore_v1.collection.CollectionReference

File: cloud_functions/climateiq_spatialize_chunk_predictions_cf/main_test.py [hunk 2]
```diff
@@ -1,18 +1,20 @@
 import base64
 import tempfile
+import io
 
+from contextlib import redirect_stdout
```
Comment (sylmak) [inline cloud_functions/climateiq_spatialize_chunk_predictions_cf/main_test.py]: Nit: Import the module, not the function: https://google.github.io/styleguide/pyguide.html#22-imports

File: cloud_functions/climateiq_spatialize_chunk_predictions_cf/requirements.txt
Comment (sylmak) [inline cloud_functions/climateiq_spatialize_chunk_predictions_cf/requirements.txt:1]: You mentioned you were tweaking this file to try to fix the mypy errors - can any of those changes be reverted?

**Review by sylmak — APPROVED** [review]

File: cloud_functions/climateiq_spatialize_chunk_predictions_cf/main.py (hunk 1 above)
Comment (skeshive) [reply cloud_functions/climateiq_spatialize_chunk_predictions_cf/main.py:211]: study_area_ref.collection returns type Any

**Pull request merged**